scripts/generate_json_api.py

```diff
@@ -369,6 +404,11 @@ class Examples(CommandTestCase):
 
         # abandon all the things
 
+        await r(
+            'Abandon a comment',
+            'comment', 'abandon', comment['comment_id']
+        )
```

Comment:
Didn't this comment already get abandoned earlier?

Reply:
different one